### DISCUSSION TITLE
PieChart component for @mantine/charts

### DISCUSSION THREAD

**Discussion by okanfidan** [mantinedev/mantine#5520] [Feature requests]
Can you add component to @mantine/charts piecharts feature in recharts? 
[PieChart - Recharts](https://recharts.org/en-US/api/PieChart)

![image](https://github.com/mantinedev/mantine/assets/6125217/3c774405-1342-44d5-bf09-0e9538cacbc7)

**Comment by SerRat44** [comment]
was coming here to ask this
